src/annotation.js

```diff
@@ -156,7 +156,7 @@ export class Annotation extends Modifier {
 
     // We're changing context parameters. Save current state.
     this.context.save();
-    const classString = Object.keys(this.getAttribute('classes')).join(' ');
+    const classString = Object.keys(this.classes).join(' ');
```

Comment:
Why not create a separate PR for this? This has nothing to do with migration to typescript, Ricardo

Reply:
If the result of `this.getAttribute('classes')` is the same as `this.classes`, it looks like an obvious simplification.

Reply:
I think that we have to as far as possible avoid the usage of `any` in order to get the advantages of TypeScript type checking and double types in order to simplify the code.

With that we should improve maintainability and we will also have the possibility to find bugs in this migration journey.

Reply:
@rvilarl did you comment on the wrong change? there is no `any` in this change.

Reply:
@sschmidTU we were writing at the same time, I have put the quote :)

Reply:
Still, i don't see what changing `this.getAttribute('classes')` to `this.classes` has to do with `any`. But i'm not against the change anyways.

Reply:
@sschmidTU have a look where I left you a question. I will expand on the topic any editing it.

Reply:
I suggest to close this conversation to leave have all the discussion in the first instance of `this.classes` in `annotation`